server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -277,6 +278,11 @@
   private static final long TIME_BETWEEN_GC_CHECKS = 5000;
   private static final long TIME_BETWEEN_LOCATOR_CACHE_CLEARS = 60 * 60 * 1000;
 
+  private static final String INGEST_COUNT = "INGEST COUNT";
```

Comment:
We are wondering if the counts will actually mean anything without knowing when the tablet was initially created on this server.  If the goal is to know the busiest tablets, then the rates are probably sufficient.  However if you want to know the total rate then we will need to log the count with along with the time that the count started.